.travis.yml

```diff
@@ -26,6 +30,12 @@ matrix:
   - env: CMD="mill demo.fullOpt && sbt readme/run"
     jdk: oraclejdk8
 
-script:
+  - stage: release
+    script:
+      - echo "$PGP_SECRET" | base64 --decode | gpg --import
+      - mill __.publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
```

Comment:
Will this run on every merge into master or only for tag pushes?

Reply:
Every merge. But things are automatically pushed to the right repo (releases or snapshots).